### ISSUE TITLE
Showing in view pending, approved, rejected

### ISSUE THREAD

**Issue by psdes** [gravityview/GravityView#447]
Hi, Im using Gravity forms, gravity view and Approval plugin from Steve Henty.
Steve in last few days add filter to plugin to translate status from pending, approved, rejected to Pending, Approved, Rejected or you can translate them to any languages you want. 
It works perfect but on GravityView those values are not.

Code of filter is:

```
	function filter_gform_entries_field_value( $value, $form_id, $field_id, $entry ) {
		$translated_value = $value;
		if ( $field_id == 'approval_status' ) {
			switch ( $value ) {
				case 'pending' :
					$translated_value = esc_html__( 'Pending', 'gravityformsapprovals' );
					break;
				case 'approved' :
					$translated_value = esc_html__( 'Approved', 'gravityformsapprovals' );
					break;
				case 'rejected' :
					$translated_value = esc_html__( 'Rejected', 'gravityformsapprovals' );
					break;
			}
		}
		return $translated_value;
	}
}
```

Can be that done which Views on similar way?

**Comment by zackkatz** [comment]
If you're a customer, please send this question to support@gravityview.co - otherwise, we'll check it out when we have time. Thanks!

**Comment by zackkatz** [comment]
This is fixed in 1.17 when using Gravity Flow. See Issue #708
